docs/content/getting-started/configuration-overview.md

```diff
@@ -29,6 +29,10 @@ Traefik gets its _dynamic configuration_ from [providers](../providers/overview.
  
 ## The Static Configuration
 
+!!! warning "Uncompatible Static Configuration"
+    Please be aware that the old configuration of v1.7.X is NOT compatible with the v2.X config as of now.
```

Comment:
```suggestion
    Please be aware that the old configurations for Traefik v1.X are NOT compatible with the v2.X config as of now.
```

Reply:
I'd suggest also replacing `config` with `configuration` for consistency